src/utils/dig.js

```diff
@@ -61,14 +65,18 @@ export const handleDig = async ({ domain, types, short, provider }) => {
             return `${digCmd}\`\`\`\n${rowsStr}${truncStr}\n\`\`\``;
         };
 
+        const maxLength = 4096 - (cdflag ? DNSSEC_DISABLED_WARNING_MESSAGE.length : 0);
+
         // Keep adding rows until we reach Discord 4096 char limit
         for (const row of sourceRows) {
-            if (output([...finalRows, row]).length > 4096) break;
+            if (output([...finalRows, row]).length > maxLength) break;
             finalRows.push(row);
         }
 
         // Render and return final rows
-        return output(finalRows);
+        return `${output(finalRows)}${cdflag
+          ? `\n${DNSSEC_DISABLED_WARNING_MESSAGE}`
+          : ''}`;
```

Comment:
```suggestion
            ? `\n${DNSSEC_DISABLED_WARNING_MESSAGE}`
            : ''}`;
```
My bad, was on two-space auto-pilot